32blit-pico/audio.cpp

```diff
@@ -3,6 +3,25 @@
 #ifdef AUDIO_I2S
 #include "pico/audio_i2s.h"
 #define HAVE_AUDIO
+#define AUDIO_SAMPLE_FREQ 44100
+#endif
+
+#ifdef AUDIO_PWM
+#include "pico/audio_pwm.h"
+#include "hardware/pio.h"
+#define HAVE_AUDIO
+#define AUDIO_SAMPLE_FREQ 441
```

Comment:
Uh, 22050? (Though `audio_pwm` ignores you and does ~22050 anyway...)

Reply:
No idea what happened here :laughing: